information-schema/information-schema-tikv-store-status.md

```diff
@@ -40,3 +40,25 @@ DESC tikv_store_status;
 +-------------------+-------------+------+------+---------+-------+
 19 rows in set (0.00 sec)
 ```
+
+The descriptions of the columns in the `TIKV_STORE_STATUS` table are as follows:
+
+* `STORE_ID`: The ID of the Store.
+* `ADDRESS`: The address of the Store.
+* `STORE_STATE`: The identifier of the Store state, which corresponds to `STORE_STATE_NAME`.
+* `STORE_STATE_NAME`: The name of the Store state. The name is `Up`, `Offline`, or `Tombstone`.
+* `LABEL`: The label set for the Store.
+* `VERSION`: The version number of the Store.
+* `CAPACITY`: The storage capacity of the Store.
+* `AVAILABLE`: The remaining storage space of the Store.
+* `LEADER_COUNT`: The number of leaders on the Store.
+* `LEADER_WEIGHT`: The leader weights of the Store.
+* `LEADER_SCORE`: The leader scores of the Store.
```

Comment:
```suggestion
* `LEADER_WEIGHT`: The leader weight of the Store.
* `LEADER_SCORE`: The leader score of the Store.
```
Although there are multiple leaders on the store, the score is only one for each store.

Reply:
Thanks, updated.